tests/test_expression.py

```diff
@@ -1,536 +1,533 @@
-import claripy
+#pylint: disable=no-self-use
```

Comment:
Please put a space between the # and pylint. Additionally, these can be comma-seperated lists rather than multiple lines.